bcda/cli.go

```diff
@@ -40,30 +40,34 @@ func createACO(name, cmsID string) (string, error) {
 }
 
 type cclfFileMetadata struct {
-	name      string
-	env       string
-	acoID     string
-	cclfNum   int
-	perfYear  int
-	timestamp time.Time
-	filePath  string
+	name         string
+	env          string
+	acoID        string
+	cclfNum      int
+	perfYear     int
+	timestamp    time.Time
+	filePath     string
+	imported     bool
+	deliveryDate time.Time
 }
 
 type cclfFileValidator struct {
 	totalRecordCount int
 	maxRecordLength  int
 }
 
-func importCCLF0(fileMetadata cclfFileMetadata) (map[string]cclfFileValidator, error) {
-	fmt.Printf("Importing CCLF0 file %s...\n", fileMetadata.name)
-	log.Infof("Importing CCLF0 file %s...", fileMetadata.name)
+const deleteThresholdHr = 8
```

Comment:
Can you add a comment indicating exactly what this represents?  Also, since it's a constant, it should be all caps I believe.

Reply:
Looks like constants follow the usual naming format in Go, not all caps like in some languages. https://stackoverflow.com/questions/22688906/go-naming-conventions-for-const